static/src/components/__tests__/UserInformationLabel.spec.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from "@cypress/vue";
+import UserInformationLabel from "../UserInformationLabel.vue";
+import App from "../../App.vue";
+
+// Label exists
+describe("UserInformationLabel", () => {
+  it("exists", () => {
+    mount(UserInformationLabel, { props: { username: "" } });
+  });
+});
+
+// Label renders correctly and is visible
+describe("App", () => {
+  it("renders properly", () => {
+    mount(App, { probs: { username: "" } });
```

Comment:
```suggestion
    mount(UserInformationLabel, { props: { username: "" } });
```
This test seems a little fishy to me, I think you rather wanted to do this?